spec/ParseQuery.Aggregate.spec.js

```diff
@@ -379,18 +379,23 @@ describe('Parse.Query Aggregate testing', () => {
   });
 
   it_exclude_dbs(['postgres'])(
```

Comment:
This should work on postgres right?

Reply:
I am not sure. I will check.

Reply:
The group by date field works, but the test fails because the $exists is not implemented for match stage on postgres. I included an additional test only for postgres.